pylint/checkers/len.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2016 Alexander Todorov <atodorov@MrSenko.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""check for incorrect usage of len() in conditions
+"""
+import itertools
+from astroid.node_classes import Call, Const, Name, UnaryOp
```

Comment:
Elsewhere in this codebase we generally use `node_classes` directly from `astroid` (it's `__init__.py` makes them accessable).   For example:

https://github.com/PyCQA/pylint/blob/c72d1070bc1dab7d0bcb45b5fe52b8f6dd4bba5e/pylint/checkers/classes.py#L223
or
https://github.com/PyCQA/pylint/blob/master/pylint/checkers/python3.py#L68